plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/catalog/jdbc/TestIcebergJdbcCatalogConnectorSmokeTest.java

```diff
@@ -117,11 +122,32 @@ public final void destroy()
     }
 
     @Test
-    @Override
-    public void testView()
+    void testUnsupportedViewDialect()
```

Comment:
We found a query failure when executing `DROP SCHEMA ... CASCADE` against schemas with views #22758 in REST catalog. Could you add a similar test and fix it if necessary?

Reply:
Done!